server/app.js

```diff
@@ -14,6 +14,7 @@ const users = require('./routes/users');
 const interviews = require('./routes/interviews');
 const questions = require('./routes/questions');
 const executeCode = require('./routes/executeCode');
+const feedback = require('./routes/feedback');
```

Comment:
If the feedback routes have been moved to /interviews, this, line 45, and the routes/feedback.js file can all be removed